# Motor status message
std_msgs/Header header
int32 left_motor_rpm
int32 right_motor_rpm
float64 left_motor_rotations
float64 right_motor_rotations
